src/status_im/chat/models.cljs

```diff
@@ -85,7 +85,7 @@
   [chat-id {:keys [db now]}]
   (let [name (get-in db [:contacts/contacts chat-id :name])]
     {:chat-id            chat-id
-     :name               (or name (gfycat/generate-gfy chat-id))
+     :name               (or name "")
```

Comment:
how can name be falsy?

Reply:
This fn is used for all kind of chats, so after I have added a `pre` to generate-gfy chokes on it, as it's passed the `chat-id` of `#status` for example. Also I believe `name` is not used for anything but group-chats.

Reply:
if generate-gfy chokes on it that would mean there is a case where name is nil and yet chat-id is not a public-key, which scenario is that?

Reply:
public chat?

Reply:
do we know at this point which type of chat it is? if that is the case it could be better handled with cond-> and make :name optional

Reply:
maybe, but should we do it another time? this code hasn't been changed and I want to limit the changes as it's already quite a change